Library/Formula/chapel.rb

```diff
@@ -22,12 +19,15 @@ def install
     # https://gist.github.com/DomT4/90dbcabcc15e5d4f786d
     # https://github.com/Homebrew/homebrew/pull/35166
     cd libexec do
-      system "make", "all"
+      system "make"
+      system "make", "chpldoc"
     end
 
     prefix.install_metafiles
 
-    (bin/"chpl").write_env_script libexec/"bin/darwin/chpl", :CHPL_HOME => libexec
+    # Install chpl and other binaries (e.g. chpldoc) into bin/ as exec scripts.
```

Comment:
We deliberately only installed the ` chpl ` binary in the past and kept the rest sandboxed. Did something change about this release to necessitate broadening that, out of curiosity?